assets/js/modules/analytics/components/settings/SettingsEdit.js

```diff
@@ -63,39 +61,6 @@ export default function SettingsEdit() {
 		}
 	}, [ hasExistingTag, existingTag, existingTagPermission ] );
 
-	// Toggle disabled state of legacy confirm changes button.
-	useEffect( () => {
-		const confirm = global.document.getElementById( 'confirm-changes-analytics' );
-		if ( confirm ) {
-			confirm.disabled = ! canSubmitChanges;
-		}
-	}, [ canSubmitChanges ] );
-
-	const { submitChanges } = useDispatch( STORE_NAME );
-	useEffect( () => {
-		addFilter(
-			'googlekit.SettingsConfirmed',
-			'googlekit.AnalyticsSettingsConfirmed',
-			async ( chain, module ) => {
-				if ( 'analytics-module' === module ) {
-					const { error } = await submitChanges();
-					if ( error ) {
-						return Promise.reject( error );
-					}
-					return Promise.resolve();
-				}
-				return chain;
-			}
-		);
-
-		return () => {
-			removeFilter(
-				'googlekit.SettingsConfirmed',
-				'googlekit.AnalyticsSettingsConfirmed',
-			);
-		};
-	}, [] );
-
```

Comment:
See my overarching PR review comment: We cannot remove legacy fixes like this until we fully migrate away from the old architecture.